pkg/controller/gardenletdeployer/actuator.go

```diff
@@ -451,71 +445,25 @@ func (a *Actuator) reconcileSeedSecrets(ctx context.Context, obj client.Object,
 		return nil
 	}
 
-	// If backup is specified and DoNotCopyBackupCredentials feature gate is disabled,
-	// create or update the backup secret if it doesn't exist or is owned by the object.
-	var (
-		checksum     string
-		allowCopying = !utilfeature.DefaultFeatureGate.Enabled(features.DoNotCopyBackupCredentials)
-	)
-
 	// Get backup secret
 	backupSecret, originalErr := kubernetesutils.GetSecretByObjectReference(ctx, a.GardenClient, spec.Backup.CredentialsRef)
 
 	if client.IgnoreNotFound(originalErr) != nil {
 		return originalErr
 	}
 
-	if apierrors.IsNotFound(originalErr) && !allowCopying {
-		return fmt.Errorf("the configured backup secret does not exist, however the feature gate DoNotCopyBackupCredentials is enabled and shoot infrastructure credentials will not be reused: %w", originalErr)
-	}
-
-	if originalErr == nil {
-		checksum = utils.ComputeSecretChecksum(backupSecret.Data)[:8]
+	if apierrors.IsNotFound(originalErr) {
+		return fmt.Errorf("the configured backup secret does not exist and shoot infrastructure credentials will not be reused: %w", originalErr)
```

Comment:
I would personally no longer mention the "[...] and shoot infrastructure credentials will not be reused". With this PR we remove this scenario. I don't know how useful is this part for the reader. It might cause confusion for the readers  in future.

```suggestion
		return fmt.Errorf("the configured backup secret does not exist: %w", originalErr)
```